README.md

```diff
@@ -59,76 +53,90 @@ def swagger_spec do
       }
     ],
     consumes: [
-      "application/vnd.api+json"
+      "application/json"
     ],
     produces: [
-      "application/vnd.api+json"
+      "application/json"
     ],
     definitions: %{},
     paths: %{},
   }
 end
 ```
 
-`PhoenixSwagger` provides `swagger_path/2` macro that generates swagger documentation
-for the certain phoenix controller.
+`PhoenixSwagger` provides the `swagger_path/2` macro that generates `swagger` documentation that you can use in your controllers.
 
 Example:
 
 ```elixir
-use PhoenixSwagger
-
-swagger_path :index do
-  get "/users"
-  summary "Get users"
-  description "Get users, filtering by account ID"
-  parameter :query, :id, :integer, "account id", required: true
-  response 200, "Description", :Users
-  tag "users"
-end
+defmodule MyApp.UserController do
+  use MyApp.Web, :controller
+  import PhoenixSwagger
+
+  swagger_path :index do
+    get "/users"
+    summary "Get users"
+    description "Get all users"
+    response 200, "Success", :Users
+    tag "users"
+  end
+
+  def index(conn, _params) do
+    posts = Repo.all(MyApp.User)
+    render(conn, "index.json", posts: posts)
+  end
 
-def index(conn, _params) do
-  posts = Repo.all(Post)
-  render(conn, "index.json", posts: posts)
+  swagger_path :show do
+    get "/users/{id}"
+    summary "Get single user"
+    description "Get a single user by id"
+    parameter :id, :path, :integer, "account id", required: true
+    response 200, "Success", :User
+    tag "users"
+  end
+  
+  def show(conn, _params) do
+    user = Repo.get!(MyApp.User, id)
+    render(conn, "show.json", user: user)
+  end
 end
 ```
 
 The `swagger_path` macro takes two parameters:
 
 * The name of the controller action
-* `do` block containing calls into the PhoenixSwagger.Path module.
+* A `do` block containing calls into the `PhoenixSwagger.Path` module.
 
-The body of the DSL is only a thin layer of syntax sugar over a regular phoenix function pipeline.
-The example above can be re-written as:
+The body of the DSL is only a thin layer of syntax sugar over a regular `Phoenix` function pipeline.
```

Comment:
@yagooar: Was this section describing the implementation of the DSL useful for you?

It was interesting to me as I was learning Elixir, but maybe it doesn't need to be in the README, just the API docs?